extensions/amp-kaltura-player/validator-amp-kaltura-player.protoascii

```diff
@@ -32,6 +32,10 @@ tags: {  # <amp-kaltura-player>
   html_format: AMP
   tag_name: "AMP-KALTURA-PLAYER"
   requires_extension: "amp-kaltura-player"
+  attrs: {
+    name: "data-service-url"
+    mandatory: false
```

Comment:
false is the default, so no need to list it explicitly

Reply:
is removed